apps/meteor/client/views/room/components/body/composer/messageBox/MessageBox.tsx

```diff
@@ -272,7 +312,7 @@ const MessageBox = ({
 
 	const isRecording = isRecordingAudio || isRecordingVideo;
 
-	const { textAreaStyle, shadowStyle } = useAutoGrow(textareaRef, shadowRef, isRecordingAudio);
+	const { textAreaStyle, shadowStyle } = useAutoGrow(textareaRef, shadowRef, isRecordingAudio, showMarkdownPreview);
```

Comment:
Using modified `useAutoGrow` to hide textarea when users preview their messages.